src/content/deployment/subctl.en.md

```diff
@@ -129,6 +140,8 @@ and between gateway and non-gateway node combinations.
 | `--operation-timeout` `<value>`     | Operation timeout for K8s API calls (default 240)
 | `--report-dir` `<string>`           | XML report directory (default ".")
 | `--verbose`                         | Produce verbose logs during connectivity verification
+| `--only`                            | Comma separated verifications to be performed (default `connectivity,service-discovery,gateway-failover`)
```

Comment:
```suggestion
| `--only`                            | Comma separated list of specific verifications to perform
```

kind of a nit but perhaps it would be better not to list the default verifications here as it could become stale should we miss updating this as we add more. Above we state that by default all verifications are performed so I think re-wording as per my suggestion should suffice.

Reply:
Done.